documents/storage-format.md

```diff
@@ -0,0 +1,57 @@
+# Storage format
+
+This document describes storage format for events from 3 sources: _engagement estimator_, _keyboard source_, _mouse_ source. Every event must have a timestamp so it possible to restore the timeline of user's **presence/absense** from the records sequence.
+
+## Engagement estimator
+
+### Engagement level change
+
+Interaction states are described in details in [_interaction-states_ document](interaction-states.md). Here we will just assign integer codes to each of them. A record will be made only when the engagement level has changed.
+
+| 0          | 1            | 2      | 3           | 4       |
+| ---------- | ------------ | ------ | ----------- | ------- |
+| engagement | conferencing | idling | distraction | absense |
+
+```SQL
+time: TEXT ("YYYY-MM-DD HH:MM:SS.SSS");
```

Comment:
In my SQLite store PR I store the time as an integer timestamp. Also maybe it'd be better to include both the format of the SQLite tables and the files?

Reply:
Included.